assembl/static2/tests/unit/components/debate/common/post/postBody.spec.jsx

```diff
@@ -26,4 +31,21 @@ describe('PostBody component', () => {
     const result = renderer.getRenderOutput();
     expect(result).toMatchSnapshot();
   });
+
+  it('should render a html body', () => {
+    const { extracts } = fakeData;
+    const props = {
+      extracts: extracts,
+      rawHtml:
+        'You can\'t index <a href="url">the port</a> without <annotation data-state="pending" id="annotationId">programming</annotation> the wireless HTTP program! <iframe src="iframe-src"></iframe>',
+      divRef: () => {},
+      dbId: '3059',
+      replacementComponents: postBodyReplacementComponents(),
+      contentLocale: 'fr'
+    };
+    const renderer = new ShallowRenderer();
+    renderer.render(<Html {...props} />);
```

Comment:
your test is under `describe('PostBody component'` but you are testing `Html` component